boards/nxp/imx95_evk/doc/index.rst

```diff
@@ -0,0 +1,202 @@
+.. _imx95_evk:
+
+NXP i.MX95 EVK
+##############
+
+Overview
+********
+
+The i.MX95 EVK (IMX95LPD5EVK-19) board is a platform designed to show the
+most commonly used features of the i.MX 95 automotive applications processor.
+It is an entry-level development board, which helps developers to get familiar
+with the processor before investing a large amount of resources in more
+specific designs. The i.MX95 device on the board comes in a compact
+19 x 19 mm package.
+
+Hardware
+********
+
+- i.MX 95 automotive applications processor
+
+  - The processor integrates up to six Arm Cortex-A55 cores, and supports
+    functional safety with built-in Arm Cortex-M33 and -M7 cores
```

Comment:
s/-M7/Cortex-M7 ?

Reply:
These are copied from i.MX95 EVK board user guide.
It's Arm if you check https://www.arm.com/
Thanks.

Reply:
It's okay to follow the Arm company. But weird, generally acronyms are made up of capital letters.

Reply:
It's okay to follow the Arm company. But weird, generally acronyms are made up of capital letters.